README.md

```diff
@@ -57,51 +54,68 @@ Run the following commands to insert two products: Avocado and Gold
 
 ```shell
 curl -X POST localhost:8080/products/create -H 'Content-Type: application/json' \
-	 -d '{"Name":"Avocado","Price":10,"Quantity":5,"_id":"11111111-1111-1111-1111-111111111111"}'
+	 -d '{"Name":"Avocado","Price":10,"Quantity":5}'
 curl -X POST localhost:8080/products/create -H 'Content-Type: application/json' \
-	 -d '{"Name":"Gold","Price":3000,"Quantity":1,"_id":"22222222-2222-2222-2222-222222222222"}'
+	 -d '{"Name":"Gold","Price":3000,"Quantity":1}'
 ```
 
 ### Place some orders
 
+In the following requests replace the placeholders with the users and products UUIDs
```

Comment:
I doubt people will read these instructions and follow it. Should we show users a way to fetch the generated ID and then use it in the rest of the example?